test/impegh_mp4_mux_utils.c

```diff
@@ -978,25 +978,88 @@ static ia_downmixInstruction* selectDownmixInstructions(ia_mpegh3daUniDrcConfig
   return NULL;
 }
 
+static void deriveDrcChannelGroups(
+  const WORD32 drcSetEffect,  /* in */
+  const WORD32 channelCount,  /* in */
+  const WORD8* gainSetIndex,  /* in */
+  UWORD8* nDrcChannelGroups,  /* out */
+  WORD8* uniqueIndex,         /* out (gainSetIndexForChannelGroup) */
+  WORD8* groupForChannel      /* out */
+)
+{
+  WORD32 duckingSequence = -1;
```

Comment:
Local variable set but not referenced anywhere in the function ?

Reply:
Removed